src/Resources/views/CRUD/base_list_field.html.twig

```diff
@@ -55,7 +55,8 @@ file that was distributed with this source code.
             ) %}
 
             {% if field_description.type == constant('Sonata\\AdminBundle\\Templating\\TemplateRegistry::TYPE_DATE') and value is not empty %}
-                {% set data_value = value.format('Y-m-d') %}
+                {# it is a x-editable format https://vitalets.github.io/x-editable/docs.html#date #}
```

Comment:
why is this repository relevant? are we using it?

Reply:
It's in our vendor https://github.com/sonata-project/SonataAdminBundle/tree/3.x/src/Resources/public/vendor/x-editable

Reply:
Yes.  We are actively using this.  See the [documentation](https://symfony.com/doc/master/bundles/SonataAdminBundle/reference/action_list.html).